script/gen-types.ts

```diff
@@ -280,10 +216,7 @@ const out = [
                               return field1.name < field2.name ? -1 : 1;
                             })
                             .map(fieldName => {
-                              const field = typeDef.allFields[fieldName];
-
-
-                              return getPropertySignature(field);
+                              return typeDef.allFields[fieldName].getPropertySignature(b);
```

Comment:
Since `Field` no longer has access to the `builders` object, I needed to pass the current `b` object into `getPropertySignature` here.